### ISSUE TITLE
Odd hole bridging bug

### ISSUE THREAD

**Issue by ptyork** [bambulab/BambuStudio#630]
A common technique to allow supportless printing of a recessed screw hole is to add a single layer, solid layer at the transition between head (or bolt) and the smaller bore for the screw itself. Generally, a slicer will add a solid base layer here that easily bridges the hole for the head/bolt. Then the screw hole is continued normally above this base layer, now fully supported. Once printed, this single layer is easily punctured for a very clean print.

Bambu handles this incorrectly. Instead of printing a base layer, it appears to perhaps start the base of the hole above it. The result is that it prints a filled circle completely unsupported.

See the comparison images. Left to right are layer before the transition, the base layer, and the subsequent start of the screw bore hole. Top to bottom are Cura, Bambu, and Prusa. Only Bambu fails, leaving an unpredicably-attached blob of plastic extruded below a now poorly formed screw hole.

![Hole_Bridging_Cura_v_Bambu_v_Prusa](https://user-images.githubusercontent.com/1051795/201708030-1a2ba4a9-aacf-4bca-b1b7-115efa813036.png)

**Comment by SaltWei** [comment]
Thanks for reporting issue to us to keep improvement!

**Comment by jeanblanchard** [comment]
I was about to report the same bug. Sacrificial bridges are a pretty common modeling trick.

Attached is a reduced test-case model that I just hacked together (CC-0). It shows the issue in BambuStudio, and works fine in PrusaSlicer (I uploaded a zip because GitHub does not like 3MF files. zip).

[Sacrificial Bridge v1.zip](https://github.com/bambulab/BambuStudio/files/10015882/Sacrificial.Bridge.v1.zip)

"Bridge" in BambuStudio: 
![Screenshot_20221115_211301](https://user-images.githubusercontent.com/760414/202016384-edec5dcc-3d19-4d65-8cf4-1d25c647d913.png)

Bridge in PrusaSlicer
![Screenshot_20221115_211404](https://user-images.githubusercontent.com/760414/202016497-8245dbda-9535-4e92-b715-2dcc970ada5b.png)
